src/libs/ajax.test.js

```diff
@@ -315,4 +315,45 @@ describe('Ajax tests', () => {
       expect(response.runs.length).toEqual(1)
     })
   })
+
+  it('should successfully POST a method', async () => {
+    const expectedResponse = {
+      run_set_id: fromProviderState('${run_set_id}', '00000000-0000-0000-0000-000000000000'), // eslint-disable-line no-template-curly-in-string
+      method_id: fromProviderState('${method_id}', '00000000-0000-0000-0000-000000000000') // eslint-disable-line no-template-curly-in-string
+    }
+
+    const payload = {
+      method_name: 'scATAC-imported-4',
+      method_source: 'GitHub',
+      method_version: 'imported-version-4',
+      method_url: 'https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/scATAC/scATAC.wdl'
+    }
+    const body = JSON.stringify(payload)
+    const headers = { 'Content-Type': 'application/json' }
+
+    await cbasPact.addInteraction({
+      states: [
+        { description: 'ready to fetch myMethodVersion with UUID 90000000-0000-0000-0000-000000000009' },
+        { description: 'cromwell initialized' }
```

Comment:
note that these states don't exist right now. this array should be updated as part of https://broadworkbench.atlassian.net/browse/WM-2010